src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -153,9 +158,9 @@ private void Analyze(TOptions analyzeOptions, AggregatingLogger logger)
             IEnumerable<Skimmer<TContext>> skimmers,
             ISet<string> disabledSkimmers)
         {
-            options.ThreadCount = options.ThreadCount > 0
-                ? options.ThreadCount
-                : Environment.ProcessorCount;
+            options.ThreadCount = options.ThreadCount > 0 ? 
```

Comment:
>ThreadCount [](start = 20, length = 11)

with this change, if no thread count is explicitly specified, we drop the thread count to one if the debugger is attached. generally, if debugging, you don't want multithreading enabled. you can still force this when debugging by providing an explicit thread count > 1.